.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "git.ignoreLimitWarning": true
```

Comment:
what’s this for? generally editor things go in global/user config rather than in each repo

Reply:
oop my bad, this was automatically added by VS Code and I didn't catch it